packages/react-native-editor/ios/DocumentsMediaSource.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+import Gutenberg
+import MobileCoreServices
+
+class DocumentsMediaSource: NSObject {
+    private var mediaPickerCallback: MediaPickerDidPickMediaCallback?
+    private unowned var gutenberg: Gutenberg
+    private unowned var uploadCoordinator: MediaUploadCoordinator
+
+    init(gutenberg: Gutenberg, coordinator: MediaUploadCoordinator) {
+        self.gutenberg = gutenberg
+        self.uploadCoordinator = coordinator
+    }
+
+    func presentPicker(origin: UIViewController, filters: [Gutenberg.MediaType], multipleSelection: Bool, callback: @escaping MediaPickerDidPickMediaCallback) {
```

Comment:
`Filters` is the name we use in all the media components. I do agree that it might be confusing and `allowedMediaTypes` would be a better fit, but I'm thinking this might require a refactor of all media blocks to do it right. Thinking that this bit was copied from WPiOS that handles all media interactions in a similar way.

Reply:
I see, thanks for the context. I'll mark this as resolved then.